src/main/java/com/infamous/combat_next/mixin/PlayerMixin.java

```diff
@@ -58,6 +60,11 @@ private float bypassZeroDamage(float constant, DamageSource source, float damage
         }
     }
 
+    @ModifyExpressionValue(method = "attack", at = @At(value = "INVOKE", target = "Lnet/minecraft/world/entity/player/Player;getAttackStrengthScale(F)F", ordinal = 0))
+    public float nullTime(float original) {
+        return MeleeCombatConfigs.getAttackCooldownImpactOnDamage().get() ? original : 1;
```

Comment:
So if you saw the comment I made in CombatUtil about this, feel free to ignore it if it is indeed meant to be used for multiple damage calculations apart from sweep.

Reply:
yes, it is. this mixin shows that